README.md

```diff
@@ -40,90 +40,33 @@ An easy way to get the full Panoptes stack running (see `fig.yml` to dig into th
   * [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) - Docker
   * [Windows](http://docs.docker.com/installation/windows/) - Boot2Docker
 
-#### Installation
+* [fig](http://fig.sh)
 
-1. Ensure your repo directory starts with a lowercase letter - you may need to move `/Panoptes` to `/panoptes`.
+#### Usage
 
-2. Prepare your fig development environment config files. You should only have to do this before the first boot. **Note:** The fig docker environment uses linked docker containers, so your Postgres and Zookeeper hosts urls need to refer to these containers.
+1. Setup the application configuration files by running `rake configure FIG_RAKE=off`
```

Comment:
What do you think about reversing the behaviour of FIG_RAKE, i.e. having it to be off by default and forcing the user to understand and opt in? Our use is only for dev and we could easily set and env variable to activate it in dev mode.